app/javascript/src/components/editor/CodeMirror.svelte

```diff
@@ -86,6 +87,19 @@
     })
   }
 
+  function handleTransactionDocChanged(view, transaction) {
+    if (transaction.docChanged) {
+      // Only perform this function if transaction is of an expected type performed by the user
+      // to prevent infinite loops on changes made by CodeMirror.
+      const userEvents = transaction.transactions.map(tr => tr.annotation(Transaction.userEvent))
+      if (userEvents.every(eventType => ["input.complete"].includes(eventType))) {
```

Comment:
I was a bit lazy previously, but this notation can now just be 
```suggestion
      if (userEvents.every(eventType => eventType === "input.complete")) {
```